### ISSUE TITLE
Allow boolean config variables

### ISSUE THREAD

**Issue by mgaitan** [cookiecutter/cookiecutter#126]
`cookiecutter.json` allows a key-value configuration, where value is a string. I would like to ask for a boolean (yes/no) variable to condition a block like this

``` jinja

    {% if use_that_magic %}
         ....
    {% endif %}
```

The easiest way should be autodetect the variable type from the default value. For example 

``` json

{ "full_name": "Martín Gaitán",
  "use_that_magic": true,  
   ...
}
```

When prompt for `use_that_magic` the valid answer should be yes or no. 

Also, could be an attribute `type` when implement #30

**Comment by myyk** [comment]
+1 boolean support would be nice.

**Comment by hackebrot** [comment]
This would be possible with #848

**Comment by tedmiston** [comment]
I'm currently working around lack of boolean support by using a choice variable with strings (`["yes", "no"]`) and `==` string comparisons in Jinja.  This might help others who wind up here looking for bool support.

**Comment by tedmiston** [comment]
Another reasonable option, perhaps cleaner, is to use 0 and 1 with Jinja's int filter.  For example...

cookiecutter.json:

```json
{
  "my_bool": 0,
}
```

or 

```json
{
  "my_bool": 1,
}
```

A file in your cookiecutter template:

```jinja
{% if cookiecutter.my_bool|int %}
  my_bool is truthy
{% else %}
  my_bool is falsy
{% endif %}
```

**Comment by jamb0ss** [comment]
so is it possible to use bool values with Python actually?
it seems that cookiecutter treats everything like strings, e.g.
`cookiecutter.json` config:
```
{
    "bool_var": false,
}
```
will promt like:
```
bool_var [False]: true
```
will result in `true` in my source code (`{{cookiecutter.bool_var}}`), not `True`
am I missing something?

**Comment by ThatXliner** [comment]
Bump.

**Comment by alkatar21** [comment]
@ThatXliner I Think this is a duplicate of #1625 which is already implemented with #1626, but not in an official release yet?
